app/views/patients/_patient.html.erb

```diff
@@ -74,6 +74,35 @@
             </div>
           <% end %>
         </div>
+        <div class="level">
+          <div class="level-left">
+            <% if patient.lab_results.present? %>
+              <div class="level-item">
+                <p class="title">Lab Results</p>
+              </div>
+            <% end %>
+          </div>
+        </div>
+      <div class="tile is-ancestor">
+        <% patient.lab_results.each do |i| %>
+          <div class="tile is-parent">
+            <div class="tile is-child box">
+              <div class="block">
+                <span class="mr-2 has-text-weight-bold">Result:</span><span class="has-text-weight-bold"><%= i.result %></span>
+              </div>
+              <div class="block">
+                <span class="mr-2 has-text-weight-bold">Code:</span><span class="has-text-weight-bold"><%= i.code %></span>
+              </div>
```

Comment:
Same comment as in `app/views/health_cards/show.pdf.erb` about using the methods for the text names here instead of the codes, as well consistency in the order that the code and result appear on the page.